### PR TITLE
Expose BMS on ergoCubSN000 via `battery_nws_yarp`

### PR DESCRIPTION
**Pull request by martinaxgloria**
I put this PR in draft since I have to check the `protocol` and `firmware` versions and test those files on ergoCub.

### CODE REVIEW

File: ergoCubSN000/hardware/battery/battery.xml [hunk 1]
```diff
@@ -4,6 +4,7 @@
 <device xmlns:xi="http://www.w3.org/2001/XInclude" name="canbattery" type="embObjBattery">
 
     <xi:include href="../../general.xml" />
+    <xi:include href="../electronics/left_arm-eb2-j0_1-eln.xml" />
```
Comment (Nicogene) [inline ergoCubSN000/hardware/battery/battery.xml:7]: Why it is called `left_arm...` ?

Comment (martinaxgloria) [reply ergoCubSN000/hardware/battery/battery.xml:7]: Talking with @AntonioConsilvio, he said me that the net board is the ems that controls the left arm (i.e. `eb2`)

**Comment by martinaxgloria** [comment]
Today @AntonioConsilvio helps me with the battery to be plugged and I succeeded in reading the values via `yarp read ... /ergocub/battery/data:o`. From the [README.md](https://github.com/robotology/icub-main/blob/7a20b62d96255fa10260c931c4fdedd06f408648/src/libraries/icubmod/embObjBattery/README.md) of the embObjBattery library, I got how to read the output:

90b71efa-be1b-456d-9e03-657b1f2ebaef

With the battery on and the robot powered with the power supply, I got these values:

de837a51-098d-4786-87c3-0190f780ced6

While without the power supply, I got:

12c4d686-10d5-4154-8869-d10d60c5cca6

I checked the charge value with the display on the ergoCub back and there I read 83 instead of 93. The other difference is in the current value which is 0 in the first case and a negative value in the second. 

cc @Nicogene @pattacini

**Review by pattacini — APPROVED** [review]

**Comment by pattacini** [comment]
@AntonioConsilvio, you may consider copying out this structure into the newborn robot ergoCubSN001.

**Comment by pattacini** [comment]
@maggia80, we have now the BMS publishing data over YARP on ergoCubSN000.
Thanks @martinaxgloria!

**Pull request merged**